assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -393,7 +410,10 @@ private void configureMultiUserMode(
     bind(org.eclipse.che.multiuser.permission.logger.LoggerServicePermissionsFilter.class);
 
     bind(org.eclipse.che.multiuser.permission.workspace.activity.ActivityPermissionsFilter.class);
-    bind(AdminPermissionInitializer.class).asEagerSingleton();
+
+    if (!Boolean.parseBoolean(System.getenv("CHE_AUTH_NATIVEUSER"))) {
```

Comment:
Can't be moved to same `if` condinion below ?

Reply:
yes, done